### PR TITLE
UCT/RC/DC/TAG: Avoid scatter2cqe for expected tag buffs

### PR DESCRIPTION
**Pull request by brminich**
## What
Avoid offloading tag buffers which fit CQE inline. Otherwise it will be a segafult when copying data from the CQE to the GPU buffer

### CODE REVIEW

**Comment by brminich** [comment]
@hoopoepg, plz review

File: src/uct/ib/rc/accel/rc_mlx5_common.c [hunk 1]
```diff
@@ -911,7 +911,7 @@ static void uct_rc_mlx5_tag_query(uct_rc_mlx5_iface_common_t *iface,
     iface_attr->cap.tag.rndv.max_iov         = 1;
     iface_attr->cap.tag.recv.max_zcopy       = port_attr->max_msg_sz;
     iface_attr->cap.tag.recv.max_iov         = 1;
-    iface_attr->cap.tag.recv.min_recv        = 0;
+    iface_attr->cap.tag.recv.min_recv        = UCT_IB_MLX5_CQE128_MAX_INL;
```
Comment (yosefe) [inline src/uct/ib/rc/accel/rc_mlx5_common.c:914]: need to check current sc2cqe configuration:
if user disabled scatter2cqe, it should be 0

File: src/uct/ib/rc/accel/rc_mlx5_common.c [hunk 2]
```diff
@@ -911,7 +911,7 @@ static void uct_rc_mlx5_tag_query(uct_rc_mlx5_iface_common_t *iface,
     iface_attr->cap.tag.rndv.max_iov         = 1;
     iface_attr->cap.tag.recv.max_zcopy       = port_attr->max_msg_sz;
     iface_attr->cap.tag.recv.max_iov         = 1;
-    iface_attr->cap.tag.recv.min_recv        = UCT_IB_MLX5_CQE128_MAX_INL;
+    iface_attr->cap.tag.recv.min_recv        = iface->super.super.config.max_inl_cqe[UCT_IB_DIR_RX],
```
Comment (yosefe) [inline src/uct/ib/rc/accel/rc_mlx5_common.c:914]: `iface->super.super.config.max_inl_cqe[UCT_IB_DIR_RX] + 1`

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by brminich** [comment]
issue is #5192 

bot:pipe:retest

File: src/uct/ib/rc/accel/rc_mlx5_common.c [hunk 3]
```diff
@@ -911,7 +911,8 @@ static void uct_rc_mlx5_tag_query(uct_rc_mlx5_iface_common_t *iface,
     iface_attr->cap.tag.rndv.max_iov         = 1;
     iface_attr->cap.tag.recv.max_zcopy       = port_attr->max_msg_sz;
     iface_attr->cap.tag.recv.max_iov         = 1;
-    iface_attr->cap.tag.recv.min_recv        = iface->super.super.config.max_inl_cqe[UCT_IB_DIR_RX],
+    iface_attr->cap.tag.recv.min_recv        =
+                       iface->super.super.config.max_inl_cqe[UCT_IB_DIR_RX] + 1,
```
Comment (yosefe) [inline src/uct/ib/rc/accel/rc_mlx5_common.c:915]: should be `;` in end of line, not `,`

**Pull request merged**